vtr_flow/scripts/python_libs/vtr/error.py

```diff
@@ -0,0 +1,32 @@
+class VtrError(Exception):
+    """
+    Base class for VTR related exceptions
+
+    Attributes:
+        msg -- An explanation of the error
+    """
+    def __init__(self, msg=""):
```

Comment:
In these cases it's nice to do something like this, with variable arguments, so that you can build strings with multiple arguments like when you call print.  Make it nice so you don't have to convert everything to str, etc.

def error(*msg):
    self.msg =  " ".join(str(item) for item in msg)

Reply:
this has been fixed